### PR TITLE
Feat/safe transfer from

### PR DESCRIPTION
**Pull request by bzuker**

### CODE REVIEW

**Review by AnnRiera — DISMISSED** [review]
LGTM!

File: src/hrc721.spec.ts [hunk 1]
```diff
@@ -71,4 +74,48 @@ describe('HarmonyProvider', () => {
...
   })
+
+  describe('transferFrom', () => {
+    it('should throw if there is no signer', () => {
+      expect(provider.transferFrom(TEST_ADDRESS_1, RESULT_TEST_ADDRESS, '1')).to.be.rejectedWith(Error)
+    })
+
+    it('should transfer the ownership of a token from one address to another', async () => {
+      const owner = await provider.ownerOf('5')
+      expect(owner).to.be.oneOf([TEST_ACCOUNT_2.address, TEST_ACCOUNT_3.address])
```
Comment (jarcodallo) [inline src/hrc721.spec.ts:85]: put all the expects at the end.

File: src/hrc721.spec.ts [hunk 2]
```diff
@@ -71,4 +74,48 @@ describe('HarmonyProvider', () => {
...
+      const receiverAccount = [TEST_ACCOUNT_2, TEST_ACCOUNT_3].find((account) => account.address !== owner)
+      if (!ownerAccount || !receiverAccount) throw new Error('Account not found')
+
+      provider.setSignerByPrivateKey(ownerAccount.privateKey)
+      const result = await provider.transferFrom(ownerAccount.address, receiverAccount.address, '5')
+
+      expect(result.txStatus).to.eq(TxStatus.CONFIRMED)
+      expect(result.receipt).to.exist
+      expect(result.receipt?.blockHash).to.be.string
+    }).timeout(30000)
```
Comment (jarcodallo) [inline src/hrc721.spec.ts:97]: this is not need it. Add a greater timeout in the mocha command like [this](https://github.com/blockcoders/near-rpc-providers/blob/main/package.json#L17-L18).

File: src/hrc721.spec.ts [hunk 3]
```diff
@@ -71,4 +74,48 @@ describe('HarmonyProvider', () => {
...
+      const receiverAccount = [TEST_ACCOUNT_2, TEST_ACCOUNT_3].find((account) => account.address !== owner)
+      if (!ownerAccount || !receiverAccount) throw new Error('Account not found')
+
+      provider.setSignerByPrivateKey(ownerAccount.privateKey)
+      const result = await provider.safeTransferFrom(ownerAccount.address, receiverAccount.address, '5')
+
+      expect(result.txStatus).to.eq(TxStatus.CONFIRMED)
+      expect(result.receipt).to.exist
+      expect(result.receipt?.blockHash).to.be.string
+    }).timeout(30000)
```
Comment (jarcodallo) [inline src/hrc721.spec.ts:119]: same

**Review by jarcodallo — CHANGES_REQUESTED** [review]

**Review by jarcodallo — APPROVED** [review]
LGTM

**Pull request merged**